Windows HostProcess Container Documentation (#28413)
kubernetes/website: content/en/docs/concepts/workloads/pods/_index.md

## Privileged mode for containers

> Any container in a Pod can enable privileged mode, using either the `privileged` (Linux) or `hostProcess` (Windows) flag on the [security context](/docs/tasks/configure-pod-container/security-context/) of the container spec. This is useful for containers that want to use operating system administrative capabilities such as manipulating the network stack or accessing hardware devices.

Reviewer: AIUI this is only valid for clusters with `WindowsHostProcessContainers` enabled.

Reviewer 2: I think we will need to have a separate wording here for Windows and Linux.  At this time, For Windows _every_ container must be hostprocess not any container.  The security context is slight different for windows as well (under the windowsOptions)

```
  securityContext:
    windowsOptions:
      hostProcess: true
```

Reviewer 3: Yes, the feature is alpha. Should we update to or `hostPorcess` (Windows) flag _if WindowsHostProcessContainers feature is enabled for the cluster_?

> For Windows clusters with the `WindowsHostProcessContainers` feature enabled, administrators can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the `windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used to perform administrative tasks as is done with Linux privileged containers.

Reviewer: ```suggestion
If your cluster has the `WindowsHostProcessContainers` feature enabled, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the `windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used to perform administrative tasks as is done with Linux privileged containers.
```
(you don't need a Windows node to create the Pod, although it won't be much use until some Windows node appears)

However, let's also mention node OS in this page, ahead of this new section, so that this isn't the first mention of Windows. Please feel free to reply to this comment if you want help with how to word that.
